src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertProjectConverterStepOptions.cs

```diff
@@ -9,5 +10,7 @@ public class TryConvertProjectConverterStepOptions
     {
```

Comment:
I know this was already here, but it feels like a mouthful for the name. Maybe just `TryConvertOptions`?